### PR TITLE
CI: disable symf in agent integration tests

### PR DESCRIPTION
**Pull request by olafurpg**
Previously, symf was enabled in agent integration tests. We have observed quite a lot of unstable test failures that seem to be related to symf so this PR disabled symf entirely in the agent integration tests to see if it stablizes the CI.

## Test plan

Non-flaky green CI.

### CODE REVIEW

File: agent/src/index.test.ts [hunk 1]
```diff
@@ -644,14 +644,14 @@ describe('Agent', () => {
             })
             const reply = await client.sendSingleMessageToNewChat(
                 'What is the name of the function that I have selected? Only answer with the name of the function, nothing else',
-                { addEnhancedContext: true }
+                { addEnhancedContext: false }
```
Comment (olafurpg) [inline agent/src/index.test.ts:647]: This test should not have enabled enhanced context in the first place. It is only asserting behavior related to the selected text and it's passing even with enhanced context enabled.

**Comment by olafurpg** [comment]
Retrying an unrelated e2e flake

```

      56 | export async function expectAuthenticated(page: Page) {
      57 |     await focusSidebar(page)
    > 58 |     await expect(page.getByText('Chat alongside your code, attach files,')).toBeVisible()
         |                                                                             ^
      59 | }
      60 |
      61 | // Selector for the Explorer button in the sidebar that would match on Mac and Linux
```

**Comment by olafurpg** [comment]
Seemed like this PR improves the stability of the non-e2e tests. I'm retrying all jobs to see if they non-e2e tests pass again 
CleanShot 2024-06-17 at 19 56 20@2x

**Comment by olafurpg** [comment]
All tests passed against except this one here
```
Error:   1) troubleshooting-authConnection.test.ts:32:3 › allow retrying on connection issues ─────────────

    Retry #1 ───────────────────────────────────────────────────────────────────────────────────────
    Error: expect(locator).toBeVisible()

    Locator: getByText('Chat alongside your code, attach files,')
    Expected: visible
    Received: <element(s) not found>
    Call log:
      - expect.toBeVisible with timeout 5000ms
      - waiting for getByText('Chat alongside your code, attach files,')


       at common.ts:58

      56 | export async function expectAuthenticated(page: Page) {
      57 |     await focusSidebar(page)
    > 58 |     await expect(page.getByText('Chat alongside your code, attach files,')).toBeVisible()
         |                                                                             ^
      59 | }
```

Retrying all jobs to as the final test to see if non-e2e tests pass again.

File: agent/src/index.test.ts (hunk 1 above)
Comment (abeatrix) [reply agent/src/index.test.ts:647]: Maybe we can add a comment to clarify this?
Previously I thought that we wanted to check if the LLM can still recognize the "selected code" we are referring to when enhanced context is included when it was set to true.

**Review by abeatrix — APPROVED** [review]

**Comment by olafurpg** [comment]
All non-e2e tests passed three times in a row (I ran "retry all" twice), so I think we have a strong signal that this PR stablizes the CI.

File: agent/src/index.test.ts (hunk 1 above)
Comment (olafurpg) [reply agent/src/index.test.ts:651]: Good point, done! To be 100% sure, I looked at the recording file and confirmed that the text selection is included in the prompt even when enhanced context is disabled 
CleanShot 2024-06-17 at 20 10 22@2x

**Comment by olafurpg** [comment]
The third time I'm seeing the same e2e test failure
```
Error:   1) troubleshooting-authConnection.test.ts:32:3 › allow retrying on connection issues ─────────────

    Retry #1 ───────────────────────────────────────────────────────────────────────────────────────
    Error: expect(locator).toBeVisible()

    Locator: getByText('Chat alongside your code, attach files,')
    Expected: visible
    Received: <element(s) not found>
    Call log:
      - expect.toBeVisible with timeout 5000ms
      - waiting for getByText('Chat alongside your code, attach files,')


       at common.ts:58

      56 | export async function expectAuthenticated(page: Page) {
      57 |     await focusSidebar(page)
    > 58 |     await expect(page.getByText('Chat alongside your code, attach files,')).toBeVisible()
         |                                                                             ^
      59 | }
      60 |
      61 | // Selector for the Explorer button in the sidebar that would match on Mac and Linux
```

File: agent/src/index.test.ts (hunk 1 above)
Comment (jtibshirani) [reply agent/src/index.test.ts:651]: @abeatrix I agree these tests were really helpful in ensuring the LLM knows what code is selected, even when there's extra enhanced context. I re-added some extra context in https://github.com/sourcegraph/cody/pull/4593.

**Pull request merged**